spring-integration-core/src/main/java/org/springframework/integration/dsl/IntegrationFlows.java

```diff
@@ -147,6 +150,30 @@ public static IntegrationFlowBuilder from(Object service, String methodName) {
 	}
 
 	/**
+	 * Provides {@link Supplier} as source of messages to the integration flow
+	 * which will be triggered by a <b>global</b> {@link SourcePollingChannelAdapter}.
```

Comment:
I think you mean "triggered by the application context's default poller (which must be present)".

There's no such thing as a global SPCA - each source gets its own SPCA.